### ISSUE TITLE
E5300 Compatibility 

### ISSUE THREAD

**Issue by allinonearz** [sebanc/brunch#1576]
I have a issue can brunch support Pentium E5300 . If yes then which Chromeos image I can use

**Comment by tejasraman** [comment]
No. There is NO brunch support for Core2Duo-era chips (chromeOS was released in 2010 and gets updates every few weeks since then, that chip was released in 2008)

Try [Chrome OS Flex ](https://www.makeuseof.com/how-to-install-chrome-os-flex-on-laptop/) - that supports your chip, but no Playstore

**Comment by KartikKumarddn** [comment]
I have tried Chrome OS flex but it also not installing

**Comment by tejasraman** [comment]
Hi @KartikKumarddn what happens? Does it boot? What cou do you have?
